src/jit/importer.cpp

```diff
@@ -4390,6 +4394,292 @@ GenTree* Compiler::impBaseIntrinsic(NamedIntrinsic        intrinsic,
             }
             break;
         }
+
+        case NI_Base_Vector256_WithElement:
+        {
+            if (!compSupports(InstructionSet_AVX))
+            {
+                // Using software fallback if JIT/hardware don't support AVX instructions and YMM registers
```

Comment:
I imagine a permute or shuffle would be better than the software fallback when the index is a constant.

Reply:
Ah, this one is just for `With`. This one is probably fine.